python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -335,39 +323,65 @@ def __get__(self, obj, cls):
         if obj is None:
             return cls._schema[name]
 
+        # If this class has an associated getter method make use of that
         if self._getter_name:
             return self._bind_method(obj, self._getter_name)()
 
         # First we want to try to get the information without prompting a load from disk
 
-        # ._data takes priority ALWAYS over ._index_cache
-        # This access should not cause the object to be loaded
-        obj_data = obj._data
-        if name in obj_data:
-            return obj_data[name]
+        obj_in_schema = name in cls._schema.datadict
```

Comment:
This change is unnecessary. Given how descriptors work and how they are assigned in our case, we can never have a case where we are in this function and have `name` not be matched to a schema item.